lib/pdk/version.rb

```diff
@@ -1,3 +1,3 @@
 module PDK
-  VERSION = '0.1.0'.freeze
+  VERSION = '0.1.20170616'.freeze
```

Comment:
This is a nitpick, and it likely doesn't matter in pre-release but I'm not a fan of this versioning. I'd rather not see the datestamp in the Y field. Would prefer something like `0.1.0-20160616`